### PR TITLE
sprint3 done

### PR DESCRIPTION
**Pull request by monsterr00**

### CODE REVIEW

File: cmd/server/storelayer/store.go [hunk 1]
```diff
@@ -1,14 +1,234 @@
...
+}
+
+func (storl *store) Close() error {
+	return storl.conn.Close()
+}
+
+func (storl *store) bootstrap(ctx context.Context) error {
+	// Проверяем, создана ли таблица
+	row := storl.conn.QueryRowContext(ctx, `
+	SELECT EXISTS (
```
Comment (arttet) [inline cmd/server/storelayer/store.go]: CHANGE REQUEST: please use migrations

**Review by arttet — CHANGES_REQUESTED** [review]

File: cmd/server/storelayer/store.go (hunk 1 above)
Comment (monsterr00) [reply cmd/server/storelayer/store.go]: done

File: cmd/server/storelayer/store.go [hunk 2]
```diff
@@ -1,14 +1,209 @@
...
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"log"
+	"time"
+
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database/postgres"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
```
Comment (arttet) [inline cmd/server/storelayer/store.go:12]: Please sort these imports

File: db/migrations/000001_create_metrics_table.up.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS metrics (
```
Comment (arttet) [inline db/migrations/000001_create_metrics_table.up.sql:1]:
```suggestion
CREATE TABLE metrics (
```

File: db/migrations/000001_create_metrics_table.up.sql [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+	ID varchar(255),
+	MType varchar(255),
```
Comment (arttet) [inline db/migrations/000001_create_metrics_table.up.sql:3]: It should be `enum`

File: internal/helpers/helpers.go [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+)
+
+func AbsolutePath(pathStart string, pathEnd string) string {
+	cwd, err := os.Getwd()
+	if err != nil {
+		log.Printf("Server: cant get rooted path")
+	}
+
+	//var filePath string
+	if path.Base(cwd) == "metric-service" { //проверяем, запущено из корня бинарником тестов или нет
```
Comment (arttet) [inline internal/helpers/helpers.go:17]: This is a strange function. Please remove it.

**Review by arttet — APPROVED** [review]

**Pull request merged**